packages/react-components/react-motion/library/src/hooks/useAnimateAtoms.ts

```diff
@@ -1,8 +1,11 @@
 import * as React from 'react';
 import type { AnimationHandle, AtomMotion } from '../types';
 
+// eslint-disable-next-line no-restricted-globals
+const win = typeof window !== undefined ? window : undefined;
```

Comment:
can you create a test for this? something like https://github.com/microsoft/griffel/blob/main/packages/react/src/utils/canUseDOM-node.test.ts